From c540072d453796898bfb6b6529059ef9915c0d3f Mon Sep 17 00:00:00 2001
From: chenyifu <chenyf@rock-chips.com>
Date: Wed, 31 Dec 2014 15:10:03 +0800
Subject: [PATCH] rk mipi: disable non-continued function of mipi clock

Not all the mipi panel support the non-continued function.
So it is better not support this function in SDK. We can
offer single patch for them who need this function.

Signed-off-by: chenyifu <chenyf@rock-chips.com>
---
 drivers/video/rockchip/transmitter/rk32_mipi_dsi.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/video/rockchip/transmitter/rk32_mipi_dsi.c b/drivers/video/rockchip/transmitter/rk32_mipi_dsi.c
index a7f5bd1cb8e6..35cedb848946 100755
--- a/drivers/video/rockchip/transmitter/rk32_mipi_dsi.c
+++ b/drivers/video/rockchip/transmitter/rk32_mipi_dsi.c
@@ -902,7 +902,7 @@ static int rk32_mipi_dsi_host_init(struct dsi *dsi)
 	rk32_dsi_set_bits(dsi, 0, phy_tx_triggers);
 	
 	/* enable non-continued function */
-	rk32_dsi_set_bits(dsi, 1, auto_clklane_ctrl);
+	/* rk32_dsi_set_bits(dsi, 1, auto_clklane_ctrl); */
 	/*
 	rk32_dsi_set_bits(dsi, 1, phy_txexitulpslan);
 	rk32_dsi_set_bits(dsi, 1, phy_txexitulpsclk);
-- 
2.35.3

